src/olympia/devhub/tests/test_views_submit.py

```diff
@@ -392,6 +395,27 @@ def test_with_source(self):
             response, reverse('devhub.submit.details', args=[addon.slug]))
         assert addon.current_version.source
         assert Addon.objects.get(pk=addon.pk).needs_admin_code_review
+        mode = oct(os.stat(addon.current_version.source.path)[stat.ST_MODE])
+        assert mode == '0100644'
```

Comment:
Note, before this patch this depended on the system umask, so it might have been passing in some envs. It wasn't in mine using docker, at least.